### PR TITLE
adding thank you page

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
R?

@alicoding Feel free to do a review on this if you get to it too.

File: public/thank-you.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta content="text/html; charset=utf-8" http-equiv="content-type">
+
+  <title></title>
```
Comment (jbuck) [inline public/thank-you.html]: Not even "Thank you"?

File: public/thank-you.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta content="text/html; charset=utf-8" http-equiv="content-type">
```
Comment (jbuck) [inline public/thank-you.html]: This shouldn't be needed

File: public/thank-you.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+  <title></title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```
Comment (jbuck) [inline public/thank-you.html]: Is jquery being used for anything here?

File: public/thank-you.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+              target="_blank">Donate</a>
+            </li>
+          </ul>
+        </div>
+      </footer>
+    </div>
+  </div><br>
+
+  <div id="signupfooter">
+    <script src="https://sendto.mozilla.org/page/-/protect-net-neutrality/js/parsley.min.js"></script>
```
Comment (jbuck) [inline public/thank-you.html]: Is this being used for anything?

File: public/thank-you.html (hunk 2 above)
Comment (ScottDowne) [inline public/thank-you.html]: It's in the main page too.

File: public/thank-you.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+</head>
+
+<body>
+  <div id="signupheader">
+
+    <div id="wrapper">
+      <header id="masthead">
+        <div class="container">
+          <a href="https://www.mozilla.org/" id="tabzilla">mozilla</a> <a href="https://www.mozilla.org" id="header-logo"><img alt="Mozilla Wordmark" class="mozilla-wordmark" src=
+          "https://sendto.mozilla.org/page/-/protect-net-neutrality/img/mozilla-wordmark.png"></a>
```
Comment (jbuck) [inline public/thank-you.html]: You should include this image in this patch locally

**Comment by ScottDowne** [comment]
@jbuck and @alicoding r? I fixed the comment to date.

File: public/thank-you.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+          </ul>
+        </div>
+      </footer>
+    </div>
+  </div><br>
+
+  <div id="signupfooter">
+    <script>
+      (function(){
+        var queryString = decodeURIComponent(window.location.search);
```
Comment (jbuck) [inline public/thank-you.html]: What are you using the names for? If they're needed, could you use a real querystring parser?

**Comment by jbuck** [comment]
@ScottDowne lgtm, with a note about removing the names

File: public/thank-you.html (hunk 6 above)
Comment (ScottDowne) [inline public/thank-you.html]: I had them so we can put them into the thank you message.

The query string parsing was just copied from the donation thank you page.

Comment (ScottDowne) [inline public/thank-you.html]: Pulling this out for now, and filing in another ticket.

**Pull request merged**